traderep/traderep.py

```diff
@@ -385,38 +385,34 @@ def __init__(self, bot):
                     await ctx.send("You closed your end of trade {} and {} {} for it. It's their turn to rep you"
                                    .format(trade_num, mod_word, partner.mention))
                 else:
-                    await ctx.send("You {} {} for trade {}.".format(
+                    await ctx.send("You {} {} for trade {} and that should mean all reps complete... 🎆Hurray!🎆".format(
                         mod_word, partner.mention, trade_num
                     ))
             else:
-                if did_close:
-                    await ctx.send("You closed your end of trade {} and {} {} for it. It's their turn to rep you..."
-                                   .format(trade_num, mod_word, partner.mention))
-                else:
-                    await ctx.send("You {} {} for trade {} and that should mean all reps complete...".format(
-                        mod_word, partner.mention, trade_num
-                    ))
+                await ctx.send("get_user_by_id failed, yell at someone! {} {} {}"
+                               .format(ctx, trade_who, self))
```

Comment:
This was never reached in my testing.

Reply:
Yeah, logic changed early in development. I think from feedback from you.